common/lib/xmodule/xmodule/x_module.py

```diff
@@ -16,6 +16,7 @@
 )
 from webob import Response
 from webob.multidict import MultiDict
+import newrelic.agent
```

Comment:
Ack. Yeah, that was leftover from the first yak I was trying to shave the night when I did this.